pacstall/__main__.py

```diff
@@ -0,0 +1,63 @@
+#!/bin/env python3
+
+"""Main Pacstall launcher file."""
+
+#     ____                  __        ____
+#    / __ \____ ___________/ /_____ _/ / /
+#   / /_/ / __ `/ ___/ ___/ __/ __ `/ / /
+#  / ____/ /_/ / /__(__  ) /_/ /_/ / / /
+# /_/    \__,_/\___/____/\__/\__,_/_/_/
+#
+# Copyright (C) 2020-2021
+#
+# This file is part of Pacstall
+#
+# Pacstall is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 3 of the License
+#
+# Pacstall is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Pacstall. If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+from fcntl import LOCK_EX, LOCK_NB, lockf
+from getpass import getuser
+from subprocess import call
+from time import sleep
+
+from rich.traceback import install
+
+from pacstall.api import message
+from pacstall.cmds import download
+from pacstall.parser import parse_arguments
+
+if __name__ == "__main__":
+    install(
+        show_locals=True
+    )  # --> Install Rich's traceback handler for better looking tracebackes
+
+    if getuser() == "root":
+        message.fancy("error", "Pacstall cannot be run as root")
```

Comment:
Should we be able to bypass this with the config file? Since a sys admin may want to run pacstall

Reply:
This will possibly be implemented in 2.1